### PR TITLE
Final PR of space-travelers-hub

### PR DESCRIPTION
**Pull request by Ambrosegithub**
###  This Pull Request contains the following changes

- Use React documentation.
- Use  React components.
- Use React props.
- Use React Router.
- Connect React and Redux.
- Handle events in a React app.
- Write unit tests with React Testing Library.
- Use styles in a React app.
- Use React life cycle methods.
- Apply React best practices and language style guides in code.
- Use store, actions, and reducers in React.
- Perform a code review for a team member.

### CODE REVIEW

File: src/Components/Missions.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/* eslint-disable */
```
Comment (Deepakdanger) [inline src/Components/Missions.js:1]: - [ ] KIndly remove this line, solve the linter manually.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { Routes, Route, BrowserRouter } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import NavBar from './Components/NavBar';
+import MyProfile from './Components/Profile/MyProfile';
+import Missions from './Components/Missions';
+import { fetchRocket } from './redux/Rocket';
+import Rocket from './Components/Rockets/Rocket';
+
+function App() {
```
Comment (Deepakdanger) [inline src/App.js:10]: - [ ] Kindly use ES6 syntax.

File: src/Components/Profile/Rockets/Rocket.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  //       myRockets.length <= 0 ? (
+  //         <div className="rocket-list">
+  //           <li className="no-rocket-title">Rockets</li>
+  //           <li className="no-rocket-text" key={0}>No Rockets available ...</li>
+  //         </div>
+  //       )
+  //         : myRockets.map((items) => <li key={items.id}>{items.rocketName}</li>)
+  //     }
+  //   </ul>
+  // </div>
```
Comment (Deepakdanger) [inline src/Components/Profile/Rockets/Rocket.js:45]: - [ ] KIndly remove the commented code.

File: src/Components/Rockets/Rocket.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { useSelector } from 'react-redux';
+import Card from './Card';
+import './Rocket.css';
+
+function Rocket() {
```
Comment (Deepakdanger) [inline src/Components/Rockets/Rocket.js:5]: - [ ] Kindly use ES6 syntax.

File: src/redux/missionFetch.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
+/** eslint-disable */
```
Comment (Deepakdanger) [inline src/redux/missionFetch.js:1]: - [ ] Kindly remove this line and solve the linter manually.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] It would be nice if the live version of the project is present.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Missions.js (hunk 1 above)
Comment (Ambrosegithub) [reply src/Components/Missions.js]: Done

File: src/Components/Profile/Rockets/Rocket.js (hunk 3 above)
Comment (Ambrosegithub) [reply src/Components/Profile/Rockets/Rocket.js]: DONE

File: src/App.js (hunk 2 above)
Comment (Ambrosegithub) [reply src/App.js]: DONE

File: src/redux/missionFetch.js (hunk 5 above)
Comment (Ambrosegithub) [reply src/redux/missionFetch.js]: DONE

File: README.md [hunk 6]
```diff
@@ -1 +1,74 @@
...
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Ambrosegithub/space-travelers-hub/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```
Comment (Rustamxon7) [inline README.md:71]: - [ ] [OPTIONAL] Please dont forget to add a link to the MIT license file 🗄in readme

**Review by Rustamxon7 — APPROVED** [review]
## Hello TEAM👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**